leap.py

```diff
@@ -1,14 +1,13 @@
 # Function definition is here
 def is_leap_year( year ):
-   if year%4 == 0 and year%100 != 0:
-       print True
+   if (year%4 == 0 and year%100 != 0) or year%400 ==0:
+       print str(year) + " is a leap year"
+   else:
+        print str(year) + " is NOT a leap year"
    return;
 
-# jak dodac else
-
-#on every year that is evenly divisible by 4
-#  except every year that is evenly divisible by 100
-#    unless the year is also evenly divisible by 400
 #For example, 1997 is not a leap year, but 1996 is.  1900 is not a leap year, but 2000 is.
-
-is_leap_year(1996)
\ No newline at end of file
+is_leap_year (1997)
```

Comment:
That space after function call is not necessary here. So `is_leap_year(1997)` instead of `is_leap_year (1997)`. See https://www.python.org/dev/peps/pep-0008/#pet-peeves ("Immediately before the open parenthesis that starts the argument list of a function call:").